SpringBoot 게시판 만들기/src/main/java/com/pironeer/week2/service/TopicService.java

```diff
@@ -0,0 +1,81 @@
+package com.pironeer.week2.service;
+
+import com.pironeer.week2.dto.request.CommentCreateRequest;
+import com.pironeer.week2.dto.request.TopicCreateRequest;
+import com.pironeer.week2.dto.request.TopicUpdateRequest;
+import com.pironeer.week2.dto.response.CommentResponse;
+import com.pironeer.week2.dto.response.TopicResponse;
+import com.pironeer.week2.repository.TopicRepository;
+import com.pironeer.week2.repository.domain.Comment;
+import com.pironeer.week2.repository.domain.Topic;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+import static java.time.LocalDateTime.now;
+
+@Service
+@RequiredArgsConstructor
+@Slf4j
+public class TopicService {
+    private final TopicRepository topicRepository;
+
+    public void save(TopicCreateRequest request){
+        Topic topic = Topic.builder()
+                        .title(request.title())
+                        .content(request.content())
+                        .createdAt(now())
+                        .updatedAt(now())
+                        .build();
+        topicRepository.save(topic);
+    }
+
+    public TopicResponse findById(Long id){
+        Topic topic = topicRepository.findById(id);
+        if(topic == null){ // 없을 경우 언체크 예외 던짐
+            throw new RuntimeException("Topic Not Found");
+        }
```

Comment:
```suggestion
        Topic topic = topicRepository.findById(id)
                .orElseThrow(() -> new RuntimeException("TOPIC NOT FOUND"));
```

orElseThrow를 활용하는 게 더 좋은 코드입니다!